internal/cli/internal/command/services/postgres/util.go

```diff
@@ -0,0 +1,51 @@
+package postgres
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"time"
+
+	"github.com/azazeal/pause"
```

Comment:
we generally try to add an empty line between 3rd party imports and local ones. Your editor should be doing this automatically, if not ping me in Slack and we'll see what's up with it.